### ISSUE TITLE
Can not add portal routes

### ISSUE THREAD

**Issue by tstauer** [sulu/sulu#5858]
| Q | A
| --- | ---
| Sulu Version | 2.2.5
| PHP Version | PHP 7.3.24
| DB Version | 5.7.30
| Browser Version | Firefox 78.8.0esr

# Actual Behavior

The file "routes_website_portal.yaml" does not exist (in: "") in routes_website_portal.yaml (which is being imported from "/var/www/project/config/routes_website.yaml"). Make sure there is a loader supporting the "portal" type.

# Expected Behavior

Load the file routes_website_portal.yaml from the routes_website.yaml using type portal.

# Steps to Reproduce

Add this to  config/routes_website.yaml:
```YAML
project:
    type: portal
    resource: "routes_website_portal.yaml"
```

Add the file routes_website_portal.yaml into the config folder, too.

Notice: As soon as the "type: portal" is removed the error is gone (but then the type portal is not considered).

**Comment by niklasnatter** [comment]
Thanks for reporting - can confirm that this is a problem of the `Sulu\Bundle\WebsiteBundle\Routing\PortalLoader`.

Unfortunately, I am not sure how to fix this. If anyone has experience with loading routes, a contribution would be highly appreciated! 

It looks like the loading works if the project contains an `AppBundle`. As a workaround, you can add such a bundle with the following changes:


```php
<?php

// src/AppBundle.php

namespace App;

use Symfony\Component\HttpKernel\Bundle\Bundle;

class AppBundle extends Bundle
{
}
```

```php
<?php

// config/bundles.php

return [
    // ...
    \App\AppBundle::class => ['all' => true],
]
```

**Comment by tstauer** [comment]
@nnatter: Thank you four your reply.

I can confirm it works by loading the routes using a bundle.

```
projectname:
    resource: "@AppBundle/Resources/config/routes_website_portal.yaml"
    type: portal
```

Unfortunately loading routes is something I'm not familiar with at all. I'd love updating that code but I could not even say which file is responsible when I looked at the issue before creating it here.

**Comment by niklasnatter** [comment]
Thanks for creating this issue! Should be fixed in #6014
